### ISSUE TITLE
Is it possible to launch centenial windows app?

### ISSUE THREAD

**Issue by Minerals14** [microsoft/WinAppDriver#708]
Actually I implement wpf app for appium test. it can connect and load UI element. but when i convert this app to UWP using Windows application packaging project ( as I know it called Centenial or Desktop Bridge app), appium can't start session.
 is there any idea to start session to Centenial App?

**Comment by yodurr** [comment]
Yeah it should work.  Did the install location of your app change?

**Comment by jamesmcroft** [comment]
Having a similar issue when trying to launch a desktop bridge application from the Windows Store using the store app ID (e.g. PackageFamilyName!App).

The application itself will launch, but the test will fail with the following error message

```
Failed to locate opened application window with appId: PackageFamilyName!App, and processId: xxxxx.
```
